Content.Client/Antag/AntagStatusIcons.cs

```diff
@@ -0,0 +1,40 @@
+using Content.Shared.StatusIcon;
+using Content.Shared.StatusIcon.Components;
+using Robust.Client.Player;
+using Robust.Shared.Prototypes;
+
+namespace Content.Client.Antag;
+
+public abstract class AntagStatusIcons<T, TB> : EntitySystem
+    where T : Component
+    where TB : Component
+{
```

Comment:
I don't think you need 2 generics for this, also needs documenting + system suffix.

Reply:
I used two because I needed a way to differentiate between regular revs and head revs since they use different icons.

Reply:
2 systems?